### DISCUSSION TITLE
Which version of Eclipse are you using?

### DISCUSSION THREAD

**Discussion by Yu-Shawn** [sine-io/cosbench-sineio#42] [Q&A] [answered]
Which version of Eclipse are you using? I can't find the Target Platform in Plug-in Development.
Could you tell me? Thanks.

**Comment by Yu-Shawn** [comment]
> Which version of Eclipse are you using? I can't find the Target Platform in Plug-in Development. Could you tell me? Thanks.

ok，I found it~

**Reply by kisscelia** [reply, marked as answer]
you can read the articles in the wiki or my [e-book](https://www.sineio.top/byte-of-cosbench/#/)
